packages/utilities/gel-foundations-styled-components/typography.js

```diff
@@ -0,0 +1,64 @@
+import { css } from 'styled-components';
+import MEDIA_QUERY_TYPOGRAPHY from '@bbc/gel-constants/breakpoints';
```

Comment:
This should be `import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-constants/breakpoints';`, since media query typography isn't the default export of the breakpoints file.